src/main/java/seedu/taskell/storage/XmlAdaptedTask.java

```diff
@@ -16,13 +16,9 @@
 public class XmlAdaptedTask {
 
     @XmlElement(required = true)
-    private String name;
-    @XmlElement(required = true)
-    private String phone;
+    private String description;
     @XmlElement(required = true)
     private String email;
```

Comment:
You are right, it should be removed. Apologise for the negligence. I have made the respective changes.